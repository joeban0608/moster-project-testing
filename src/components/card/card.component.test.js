import { shallow, mount, render } from "enzyme";
import React from "react";
import Card from "./card.component";
const mockMonster = {
  id: 10,
  name: "Clementina DuBuque",
  email: "Rey.Padberg@karina.biz",
};

it('is render Card component', ()=>{
  expect(shallow(<Card monster={mockMonster} />).length).toEqual(1)
})
