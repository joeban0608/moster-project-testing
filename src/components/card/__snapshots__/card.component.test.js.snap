// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is render Card component 1`] = `
<div
  className="card-container"
>
  <img
    alt="monster Clementina DuBuque"
    src="https://robohash.org/10?set=set2&size=180x180"
  />
  <h2>
    Clementina DuBuque
  </h2>
  <p>
    Rey.Padberg@karina.biz
  </p>
</div>
`;
